__tests__/app.test.js

```diff
@@ -259,7 +248,34 @@ describe("POST /api/articles/:article_id/comments", () => {
 				.then((response) => {
 					expect(response.body.msg).toEqual("Bad request");
 				});
-		});
+        });
+
+        test("responds with a 400 status when posting to an article that doesn't exist", () => {
+					const testComment = {
+						username: "rogersop",
+						body: "Wow. I, for one, am shocked and appalled!!!",
+					};
+					return request(app)
+						.post("/api/articles/9000/comments")
+						.send(testComment)
+						.expect(400)
+                        .then((response) => {
+							expect(response.body.msg).toEqual("Bad request");
+						});
```

Comment:
This shouldn't be a bad request or a 400 as 9000 is a valid number. It just doesn't exist as an article just yet. Please change it to a 404